src/main/java/checklst/exception/ChecklstException.java

```diff
@@ -0,0 +1,21 @@
+package checklst.exception;
+
+/**
+ * Custom exception to represent Exceptions in the Checklst program.
+ */
+public class ChecklstException extends Exception {
+
+    /**
+     *
+     */
+    private static final long serialVersionUID = 4447981462872622989L;
```

Comment:
Perhaps this should be 'SERIAL_VERSION_UID' as it is a constant?

Reply:
I also thought so but apparently this is some optional requirement by java and is meant to be spelled like this? I'm not too sure because my IDE (vscode) added it in automatically.